recipe/meta.yaml

```diff
@@ -188,35 +200,30 @@ outputs:
         - typing_extensions >=3.6.6
         - wrapt >=1.11.0
         # TF-API needs to move in sync
-        - tensorboard >=2.11,<2.12
-        - keras >=2.11,<2.12
+        - tensorboard >=2.12,<2.13
+        - keras >=2.12,<2.13
       run:
         - python
         - packaging
         - absl-py >=1.0.0
         - astunparse >=1.6.0
         - gast >=0.2.1,<=0.4.0
```

Comment:
FYI, it turns out that fixing https://github.com/tensorflow/tensorflow/issues/56244 was just a PR away. We could consider lifting this, but it's not critical for 2.12.